tap_shopify/__init__.py

```diff
@@ -156,6 +156,11 @@ def sync():
             except pyactiveresource.connection.UnauthorizedAccess as exc:
                 raise ShopifyError(exc, 'Invalid access token - Re-authorize the connection') \
                     from exc
+            except pyactiveresource.connection.ForbiddenAccess as exc:
+                raise ShopifyError(exc, 'Add read_inventory scope for access token and '
+                                   'Re-authorize the connection to sync '
+                                   'Inventory Levels and Inventory Items streams.') \
+                    from exc
```

Comment:
Change it to "Add read_inventory scope for access token, re-authorize the connect to sync and get Inventory levels /Inventory items streams

Reply:
Updated